swat/cas/table.py

```diff
@@ -3425,7 +3424,7 @@ def _materialize(self, casout=None, inplace=False, prefix=None, suffix=None):
         casout : dict, optional
             The CAS output table definition
         inplace : bool, optional
-            Should the output table be overwritten?
+            If True, the output table is overwritten
```

Comment:
You might want to add "if it exists" at the end

Reply:
Changed as suggested.